README.md

```diff
@@ -48,7 +48,7 @@ more. See [the wiki](https://github.com/neovim/neovim/wiki/Installing-Neovim)!
 
 ### Community
```

Comment:
Also those links up top might be easy to miss, so perhaps they should be made bold, or have their size increased a little. Really anything to make them stand out.